scripts/populate_location_data.py

```diff
@@ -46,7 +43,7 @@ def process_locations(
             postcode = get_postcode_from_questions(questions)
             application_ids_to_postcodes[id] = postcode
             just_postcodes.append(postcode)
-
+        print(just_postcodes)
```

Comment:
Do we need to print these or is that just for debugging?

Reply:
🤦‍♀️